distribution/rules.bzl

```diff
@@ -142,12 +142,34 @@ def deploy_rpm(name,
         modes = permissions,
     )
 
+    if "osx_build" not in native.existing_rules():
+        native.config_setting(
+           name = "osx_build",
+           constraint_values = [
+               "@bazel_tools//platforms:osx",
+               "@bazel_tools//platforms:x86_64",
+           ]
+        )
+
+    if "linux_build" not in native.existing_rules():
+        native.config_setting(
+            name = "linux_build",
+            constraint_values = [
+                "@bazel_tools//platforms:linux",
+                "@bazel_tools//platforms:x86_64",
+            ]
+        )
+
     pkg_rpm(
         name = name,
         architecture = "x86_64",
         spec_file = spec_file,
         version_file = rpm_version_file,
         data = [tar_name],
+        rpmbuild_path = select({
+            ":linux_build": "/usr/bin/rpmbuild",
```

Comment:
can we make it so it uses whatever `rpmbuild` binary which exists in our `$PATH` instead of specifying an absolute value like this? eg., if the user can invoke `rpmbuild` from his/her command line, Bazel will have access to it.

Reply:
@lolski we **cannot**, `bazel` forces `PATH` to be exactly `/bin:/usr/bin`